src/test/ui/rfc-2361-dbg-macro/dbg-macro-expected-behavior.rs

```diff
@@ -1,7 +1,5 @@
 // run-pass
-// ignore-cloudabi no processes
-// ignore-emscripten no processes
-// ignore-sgx no processes
```

Comment:
hm, presumably we should keep these as we're still going to be unable to run-pass test this on these platforms? It seems like a followup issue about moving these ignores directly into compiletest for all run-pass tests may make sense though.